app/src/main/res/layout/activity_main.xml

```diff
@@ -7,13 +7,195 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
-    <TextView
-        android:layout_width="wrap_content"
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
ConstraintLayout is expensive from the performance perspective so having more than one would cause some performance issues

Reply:
Change ConstraintLayout to View